diff --git a/node_modules/jsonwebtoken/index.js b/node_modules/jsonwebtoken/index.js
index 161eb2d..8cee4ce 100644
--- a/node_modules/jsonwebtoken/index.js
+++ b/node_modules/jsonwebtoken/index.js
@@ -1,8 +1,6 @@
-module.exports = {
-  decode: require('./decode'),
-  verify: require('./verify'),
-  sign: require('./sign'),
-  JsonWebTokenError: require('./lib/JsonWebTokenError'),
-  NotBeforeError: require('./lib/NotBeforeError'),
-  TokenExpiredError: require('./lib/TokenExpiredError'),
-};
+module.exports.decode = require('./decode');
+module.exports.verify = require('./verify');
+module.exports.sign = require('./sign');
+module.exports.JsonWebTokenError = require('./lib/JsonWebTokenError');
+module.exports.NotBeforeError = require('./lib/NotBeforeError');
+module.exports.TokenExpiredError = require('./lib/TokenExpiredError');
